components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -2605,17 +2599,14 @@ def getGroupsMemberOf(self):
         :rtype:         :class:`ExperimenterGroupWrapper` generator
         """
 
-        q = self.getQueryService()
-        p = omero.sys.Parameters()
-        p.map = {}
-        p.map["ids"] = rlist([rlong(a)
-                              for a in self.getEventContext().memberOfGroups])
-        sql = "select e from ExperimenterGroup as e where e.id in (:ids)"
-        for e in q.findAllByQuery(sql, p, self.SERVICE_OPTS):
-            if e.name.val == "user":
```

Comment:
@aleksandra-tarkowska Probably it's simply that users are not leaders of the ```user``` group, so we never had the problem of ```user``` group showing up in that query. I think it makes sense to unify it.